apps/pie-storybook/.storybook/preview.ts

```diff
@@ -77,6 +77,7 @@ export default {
                         'Events',
                         'Design tokens cookbook',
                         'CSS variables',
+                        'Typescript usage',
```

Comment:
Nit: Can we be consistent with casing? TypeScript I think is the official spelling